airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/pipeline.py

```diff
@@ -105,11 +109,21 @@ async def _run(self, built_containers_per_platform: Dict[Platform, Container]) -
         ]
         connector_technical_name = self.context.connector.technical_name
         connector_version = self.context.metadata["dockerImageTag"]
+        connector_has_dockerfile = self.context.connector.has_dockerfile
+        connector_is_using_poetry = self.context.connector.is_using_poetry
+
+        base_image_tag = self.context.metadata.get("connectorBuildOptions", {}).get("baseImage", "")
+        base_image_version_match = re.search(r":([0-9]+\.[0-9]+\.[0-9]+)@", base_image_tag)
+        connector_base_image_version = base_image_version_match.group(1) if base_image_version_match else ""
+
         dependencies_metadata = ConnectorDependenciesMetadata(
```

Comment:
💅 This is no longer a dependencies metadata...

Reply:
Feel free to rename! 🙃